<launch>


	<arg name="fcu_url" default="udp://127.0.0.1:14551@:14551" />
	<arg name="gcs_url" default="" /> <!-- This is computer IP -->
	<arg name="tgt_system" default="1" />
	<arg name="tgt_component" default="1" />
	<arg name="log_output" default="screen" />
	<arg name="fcu_protocol" default="v2.0" />
	<arg name="respawn_mavros" default="false" />

	<include file="$(find mavros)/launch/node.launch">
		<arg name="pluginlists_yaml" value="$(find mavros)/launch/px4_pluginlists.yaml" />
		<arg name="config_yaml" value="$(find mavros)/launch/px4_config.yaml" />

		<arg name="fcu_url" value="$(arg fcu_url)" />
		<arg name="gcs_url" value="$(arg gcs_url)" />
		<arg name="tgt_system" value="$(arg tgt_system)" />
		<arg name="tgt_component" value="$(arg tgt_component)" />
		<arg name="log_output" value="$(arg log_output)" />
		<arg name="fcu_protocol" value="$(arg fcu_protocol)" />
		<arg name="respawn_mavros" default="$(arg respawn_mavros)" />
	</include>

		<node pkg="computer_pkg" type="go_to_point_node" name="go_to_point_node" output="screen">
			<param name="x_coord" value="0.0" />
			<param name="y_coord" value="0.0" />
			<param name="z_coord" value="2.0" />
		</node>
		<node pkg="computer_pkg" type="go_to_point_gui.py" name="go_to_point_gui" output="screen"/>
		
</launch>
<!-- This is the FCU port number for real drone: 14551 -->
<!-- This is the ROS Master URI Port Number: 11311 -->
